packages/ui/src/components/editor/panels/Properties/container/index.tsx

```diff
@@ -80,7 +80,7 @@ const EntityEditor = (props: { entityUUID: EntityUUID; multiEdit: boolean }) =>
     return () => {
       window.removeEventListener('resize', handleResize)
     }
-  }, [])
+  }, [popupRef])
```

Comment:
refs inside useEffects would have the same effects as empty deps
this is because  component is rendered -> refs are set -> useEffects run (making it the same as empty deps)